### PR TITLE
Problem:(CRO-586) No way to add a node to validator set from client-rpc

### PR DESCRIPTION
**Pull request by linfeng-crypto**
solution: Add a `staking_validatorNodeJoin` method in rpc.

### CODE REVIEW

**Comment by tomtau** [comment]
bors retry

**Comment by tomtau** [comment]
bors retry

File: client-rpc/src/rpc/staking_rpc.rs [hunk 1]
```diff
@@ -47,6 +50,15 @@ pub trait StakingRpc: Send + Sync {
 
     #[rpc(name = "staking_unjail")]
     fn unjail(&self, request: WalletRequest, unjail_address: String) -> Result<String>;
+
+    #[rpc(name = "staking_validatorNodeJoin")]
+    fn node_join(
```
Comment (leejw51crypto) [inline client-rpc/src/rpc/staking_rpc.rs]: how about
"join"

**Pull request merged**